src/development/troubleshoot.md

```diff
@@ -19,7 +19,7 @@ git push platform <branch-name>
 
 ## HTTP 502 response
 
-If you receive HTTP 502 responses from your site, it means that your site is running out of workers, or the application process is crashing.  Here are the typical causes:
+If you receive HTTP 502 responses from your site, it means that your site is running out of workers or the application process is crashing.  Here are the typical causes:
```

Comment:
This additional comma is actually wrong and a comma splice. 😄   Please remove it, then go ahead and merge this PR.

Reply:
Oh, duh, I totally read that backwards for some reason.  We're good, please merge.